content/en/docs/tutorials/security/apparmor.md

```diff
@@ -118,29 +118,9 @@ gke-test-default-pool-239f5d02-xwux: kubelet is posting ready status. AppArmor e
 
 <!-- lessoncontent -->
 
-## Securing a Pod
+## Loading AppArmor profiles onto nodes
 
-{{< note >}}
-AppArmor is currently in beta, so options are specified as annotations. Once support graduates to
-general availability, the annotations will be replaced with first-class fields (more details in
-[Upgrade path to GA](#upgrade-path-to-general-availability)).
-{{< /note >}}
-
-AppArmor profiles are specified *per-container*. To specify the AppArmor profile to run a Pod
-container with, add an annotation to the Pod's metadata:
-
-```yaml
-container.apparmor.security.beta.kubernetes.io/<container_name>: <profile_ref>
-```
-
-Where `<container_name>` is the name of the container to apply the profile to, and `<profile_ref>`
-specifies the profile to apply. The `profile_ref` can be one of:
-
-* `runtime/default` to apply the runtime's default profile
-* `localhost/<profile_name>` to apply the profile loaded on the host with the name `<profile_name>`
-* `unconfined` to indicate that no profiles will be loaded
-
-See the [API Reference](#api-reference) for the full details on the annotation and profile name formats.
+*This example assumes you have already set up a cluster with AppArmor support.*
```

Comment:
We should make that clear after
```markdown
## {{% heading "prerequisites" %}}
```

then we can hyperlink back to that heading to explain what we expect the reader to have done.

Reply:
Added hyperlink in [d6abe42](https://github.com/kubernetes/website/pull/37191/commits/d6abe421da86cda4632870d5f551d9fd2b4a1654)